pageserver/src/page_service.rs

```diff
@@ -717,8 +728,9 @@ impl Connection {
             self.write_message_noflush(&BeMessage::CommandComplete)?;
             self.write_message(&BeMessage::ReadyForQuery)
         } else {
-            self.write_message_noflush(&BeMessage::RowDescription)?;
-            self.write_message_noflush(&BeMessage::DataRow)?;
```

Comment:
Let me know if we'd prefer to keep this, and maybe we can filter for the specific query that psycopg2 uses

Reply:
Should all the other command be case insensitive as well?

Reply:
Other commands are controlled by us and we can just write them in lower case. But 'SET' is part of SQL which is case-insensitive and I would not be surprised if there is some client that would call 'set' in lower register while psycopg2 calls them in upper.